### DISCUSSION TITLE
Add support for arm platforms like Android.

### DISCUSSION THREAD

**Discussion by DanixUwU** [fooyin/fooyin#427] [Ideas]
I don't know if this question/request will ever be consider, but i will try it anyway.
Resuming the situation, i wanted to use foobar2000 to experiment, but i use Linux, and everyone here probably knows that foobar2000 doesn´t support linux, so because of i started to look for alternative, and then i discovered fooyin, i started to use it, and its not bad, i am feeling more interested and comfortable with the program and so far its going grat, but the problem is that i also use Android, and this app doesn´t have a version for Android like foobar2000, and it sucks, because, i don´t really want to use 2 separate apps just to listen to music, and because of that, plus the fact that a android app would be very cool, that i am were to discuss/request the possibility of having fooyin on Android or realistically any type Os for arm device, like MacOs.
I hope that anyone that reading this understands the situation, and sorry for any type of inconvenience.

**Comment by Euphonit** [comment]
You can build fooyin on macos. Check out the pull request Instructions for macos to BUILD.md for more info.

**Comment by Palrex** [comment]
Foobar2000 android version feels very barebones when compared to the desktop version. An android version for fooyin would be awesome, and a great chance to overtake foobar2k on mobile
